libsnowflakeclient/examples/CMakeLists.txt

```diff
@@ -3,8 +3,43 @@
 # includes catch.hpp, a test framework
 include_directories(
         "../include"
+        "utils"
 )
+set(SOURCE_UTILS
+        example_setup.c)
 
-add_subdirectory(connect)
-add_subdirectory(select1)
-add_subdirectory(transaction)
+add_executable(ex_connect ${SOURCE_UTILS} connect.c)
```

Comment:
Can you please explain what the advantage of merging CMakeLists.txt here is?

Reply:
In a future pull request, this file will use a foreach loop to reduce the amount of code repeated. This way we'll just have to add the name of the example and everything will be done for us.